### ISSUE TITLE
0.4: defaultSort does not seem to work

### ISSUE THREAD

**Issue by webteckie** [keystonejs/keystone#2022]
Not sure if it's just my environment (latest chrome, windows 10) but adding `defaultSort: "name"` to a list does not seem to enable sorting on that list field.

I've added keystonejs/keystonejs-site#54 just in case it's a documentation issue.

Warning messages in console:
![image](https://cloud.githubusercontent.com/assets/5872515/12052022/c01ef382-aed8-11e5-9fd3-84cea5245ea2.png)

Used like this (according to doc example):
```
var MyList = new keystone.List('MyList', {
    defaultSort: '+name'
});
MyList.add({
    name: {
        type: String,
    },
    ...
});
```

**Comment by webteckie** [comment]
Can someone verify that default sort works (or not)?  Perhaps I'm using it wrong?  Perhaps the documentation needs more clarification?

**Comment by geloescht** [comment]
Well, `defaultSort : 'name'` works, as does `defaultSort : '-name'`. Usage with a plus sign is not mentioned in the docs as far as I can tell, but probably should be possible for the sake of completeness.

**Comment by mxstbr** [comment]
Sounds reasonable @geloescht, should take a look at this after #2566 is merged!

**Comment by geloescht** [comment]
Oops, just opened a pull request #2616

**Comment by mxstbr** [comment]
Will be fixed as soon as #2566 lands!

**Comment by sashamjolnir** [comment]
It seems like`defaultSort: '-createdAt'` don't work.
Keystone 0.4. console warns: `Invalid Sort specified: createdAt`

I copy it from http://keystonejs.com/docs/database/#lists-example. What can be wrong?
P.S. I don't want to open new issue, because this issue looks pretty familiar for my problem.

**Comment by geloescht** [comment]
This issue was about using a plus sign as a possible prefix for defaultSort.
You could help us by trying to revert your keystone module to an earlier revision and see if it works there (try one or two months ago). If it does, that sounds like a regression and you should definitely file another bug report.

**Comment by sashamjolnir** [comment]
ok, sorry, I forget to add createdAt field. example contains it, so there is no mistake. it's my fault.
